workspaces/jenkins/plugins/jenkins/report.api.md

```diff
@@ -154,6 +157,11 @@ export interface Project {
 //
 // @public (undocumented)
 export const Router: (props: {
+  title?: string;
   columns?: TableColumn<Project>[];
 }) => React_2.JSX.Element;
+
+// Warnings were encountered during analysis:
```

Comment:
Looks like there's a missing export, can you please correct this?

Reply:
The problem is that the way the code was organized, if I export this type, the entire module goes with it. To avoid this problem and the typing warning, I put the typing directly in the property instead of creating a separate type ([this commit](https://github.com/backstage/community-plugins/pull/1940/commits/bcb3b690faa552950451c5e3cafca68660496980)). Is that okay?